rust/tvm-rt/src/object/object_ptr.rs

```diff
@@ -147,14 +148,26 @@ impl Object {
     }
 }
 
+// impl fmt::Debug for Object {
+//     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+//         let index =
+//             format!("{} // key: {}", self.type_index, "the_key");
+
+//         f.debug_struct("Object")
+//          .field("type_index", &index)
+//          // TODO(@jroesch: do we expose other fields?)
+//          .finish()
+//     }
+// }
+
```

Comment:
Remove dead code.

For more information on this comment, see https://fixie.ai/comment/68314539a.

Reply:
I'm leaving the code in for now, since I plan to refactor this later.

Reply:
OK, resolving.